cddagl/ui.py

```diff
@@ -378,7 +470,7 @@ def check_new_launcher_version(self):
     def lv_http_finished(self):
         self.lv_html.seek(0)
         document = html5lib.parse(self.lv_html, treebuilder='lxml',
-            encoding='utf8', namespaceHTMLElements=False)
+            namespaceHTMLElements=False)

```

Comment:
On my python deployment (Python 3.5.1 from Homebrew) there is no such parameter as 'encoding'